checkout/access-checkout-android/src/main/java/com/worldpay/access/checkout/client/checkout/AccessCheckoutCVVClient.kt

```diff
@@ -22,19 +16,36 @@ import com.worldpay.access.checkout.views.SessionResponseListener
  * [AccessCheckoutCVVClient] is responsible for handling the request for a session state from the Access Worldpay services.
  */
 class AccessCheckoutCVVClient private constructor(
-    private val baseURL: String,
-    private val merchantID: String,
+    private val baseUrl: String,
+    private val merchantId: String,
     private val context: Context,
-    private val mListener: SessionResponseListener,
-    private val lifecycleOwner: LifecycleOwner):
-    LifecycleObserver, SessionResponseListener {
-
-    private lateinit var localBroadcastManager: LocalBroadcastManager
-    private val localBroadcastManagerFactory = LocalBroadcastManagerFactory(context)
-    private val sessionReceiver: SessionReceiver =
-        SessionReceiver(this)
+    private val externalSessionResponseListener: SessionResponseListener,
+    private val lifecycleOwner: LifecycleOwner
+) {
+
+    private val tag = "AccessCheckoutClient"
```

Comment:
Is that correct given that we are in the `AccessCheckoutCVVClient` class?

Reply:
Done